source/_integrations/epson.markdown

```diff
@@ -31,11 +34,6 @@ host:
   description: The host name or address of the Epson projector
   required: true
   type: string
-port:
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
I suggest making an additional core PR where we deprecate the configuration via configuration.yaml. Then we can remove the section here completely.

Reply:
I agree @MartinHjelmare, so what would be the steps?
PR with logger info with deprecation notice and PR for eg. 2021.7 which removes configuration.yaml?

Reply:
Log a warning in `async_setup_platform` which will go out in 2021.6 and then remove configuration.yaml support in 2021.8.